sdk/search/Azure.Search.Documents/samples/Sample07_VectorSearch_UsingReducedEmbeddings.md

```diff
@@ -98,7 +98,7 @@ public static ReadOnlyMemory<float> GetEmbeddings(string input)
 {
     Uri endpoint = new Uri(Environment.GetEnvironmentVariable("OpenAI_ENDPOINT"));
     string key = Environment.GetEnvironmentVariable("OpenAI_API_KEY");
-    AzureKeyCredential credential = new AzureKeyCredential(key);
+    ApiKeyCredential credential = new ApiKeyCredential(key);
```

Comment:
FYI @ShivangiReja, `AzureKeyCredential` will still work once Azure.Core ships this cycle ... do you want to use a project reference in Azure.Search.Documents for now?

Reply:
Thanks for the update! Now that Azure.Core has been released, I've updated `ApiKeyCredential` to `AzureKeyCredential`.